datahub-web-react/src/app/create/AdHocPage.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { CSVReader } from 'react-papaparse';
+import { Form, Input, Space, Select, Button, message } from 'antd';
+import { MinusCircleOutlined, PlusOutlined } from '@ant-design/icons';
+import adhocConfig from '../../conf/Adhoc';
+import { SearchablePage } from '../search/SearchablePage';
+
+export const AdHocPage = () => {
```

Comment:
Interesting - this page was created in order to permit ad-hoc ingestion via the UI for different entities?